src/main/java/org/verapdf/rest/resources/VerificationHealthCheck.java

```diff
@@ -0,0 +1,80 @@
+package org.verapdf.rest.resources;
+
+
+import com.codahale.metrics.health.HealthCheck;
+import io.dropwizard.setup.Environment;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.ClientProperties;
+import org.glassfish.jersey.client.JerseyClientBuilder;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.glassfish.jersey.media.multipart.FormDataMultiPart;
+import org.glassfish.jersey.media.multipart.MultiPartFeature;
+
+public class VerificationHealthCheck extends HealthCheck {
+    private final Client client;
+    private final String endpointUrl;
+
+    public VerificationHealthCheck(Environment environment, String endpointUrl) {
+        ClientConfig config = new ClientConfig();
+        config.register(MultiPartFeature.class);
+        this.client = new JerseyClientBuilder()
+                .withConfig(config)
+                .property(ClientProperties.CONNECT_TIMEOUT, 5000)
+                .property(ClientProperties.READ_TIMEOUT, 15000)
+                .build();
+        this.endpointUrl = endpointUrl;
+    }
+
+    @Override
+    protected Result check() throws Exception {
+        try {
+            Path pdfFilePath = Paths.get("test/test_document.pdf");
+            byte[] pdfBytes = Files.readAllBytes(pdfFilePath);
```

Comment:
In this case it is probably fine to `readAllBytes` at once, but in general better practice is to use streams.
Also it might be decent idea to read this file once and keep it in the memory, so that it doesn't use I/O every time the function is called.